Resources/Locale/en-US/nutrition/components/food-sequence.ftl

```diff
@@ -128,3 +128,7 @@ food-sequence-taco-gen = taco with {$content}
 # SKEWER
 
 food-sequence-skewer-gen = {$content} kebab
+
+food-sequence-skewer-content-snake = snake
```

Comment:
Use general locId
Check line 5

Reply:
done. do you know which part of the foodenings in which you removed the kebab microwave recipes / where else I might find them so I can add them as metamorphs?